base_report_to_printer/tests/test_printing_job.py

```diff
@@ -5,10 +5,12 @@
 
 from odoo import fields
 from odoo.tests.common import TransactionCase
+from odoo.tools import mute_logger
 
 model = "odoo.addons.base_report_to_printer.models.printing_server"
 
 
+@mute_logger("odoo.addons.base_report_to_printer.models.printing_server")
```

Comment:
I'm wondering if this explain the codecov complains, I'll try to move it on propoer method instead...